Calculator/Calculator/Model/LinkedList.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  LinkedList.swift
+//  Calculator
+//
+//  Created by 혜모리 on 2023/01/25.
+//
+
+final class LinkedList {
+    private(set) var head: Node?
+    private(set) var tail: Node?
+    var isEmpty: Bool {
+        head == nil
+    }
+    
+    func appendLast(_ data: String) {
+        let node = Node(value: data)
+        
+        guard !isEmpty else {
+            head = node
+            tail = head
+            return
+        }
+        tail?.next = node
+        tail = node
+    }
+    
+    func removeLast() -> Node? {
+        var currentNode = head
+        
+        guard !isEmpty else {
+            return nil
+        }
+        
+        guard head != tail else {
+            head = nil
+            return currentNode
+        }
+        
+        while currentNode?.next?.next != nil {
+            currentNode = currentNode?.next
+        }
+        
+        let lastNode = currentNode?.next
+        currentNode?.next = nil
+        tail = currentNode
+        return lastNode
+    }
+    
+    func removeFirst() -> Node? {
+        let firstNode = head
+        
+        guard !isEmpty else {
+            return nil
+        }
```

Comment:
```suggestion
        guard isEmpty == false else {
            return nil
        }
```

개인적으로 느낌표를 통해 논리부정 연산자를 사용하는 것보다는 이런 식으로 명확하게 false를 표현 해주는 것이 가독성이 더 좋다고 생각해요 . 참고만 해주세요☺️

Reply:
`isEmpty == false`로 수정했습니다! 감사합니다 😊